mimic/test/test_flavors.py

```diff
@@ -33,25 +33,23 @@ def get_server_flavor(self, postfix):
         self.assertEqual(200, response.code)
         return content
 
+    def test_get_flavor(self):
+        """
+        Test to verify :func:`get_flavor` on ``GET /v2.0/<tenant_id>/flavors/<flavor_id>``
+        """
+        get_server_flavor = self.get_server_flavor('/flavors/2')
+        self.assertEqual(get_server_flavor['flavor']['id'], '2')
+
     def test_get_server_flavor_negative(self):
         """
         Test to verify :func:`get_flavor` when invalid flavor from the
         :obj: `mimic_presets` is provided.
         """
         get_server_flavor = request(self, self.root, "GET", self.uri +
-                                    '/flavors/1')
+                                    '/flavors/100')
```

Comment:
OK, I see. Plain number IDs are used by standard flavors, but the ID can be any string. Can we use an ID that's more obviously not going to be found, such as `negative-test-1` or similar?